vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -1,29 +1,34 @@
 ---
 sidebarDepth: 2
 ---
-target=restored
-original=source
+
 # Backup and Restore an Entando Application
 
 This guide provides general instructions on how to backup and restore an existing Entando Application. The process entails the migration of databases, filesystem data, Keycloak instance, and bundles, plus the reconfiguration of the service address. 
 
 ### Initial Considerations to Note
 * The restored instance can be on another cluster or a different namespace of the same cluster hosting the original Entando instance. 
-* The source and restored instances cannot be executed simultaneously, and data cannot be shared between them.
-* Bundles are externally deployed to an Entando Application and therefore must be reinstalled after the new instance is built. More details are below.
+* The source and restored instances cannot be executed simultaneously if they use the same domain name. 
+* Data cannot be shared between the old and new instances.
+* Bundles are deployed via Kubernetes to a specific Entando Application and therefore must be reinstalled after the new instance is built. More details are below.
 
 ## Copy and Migrate Databases
 
 1. Make a copy of the source databases or schemas using the native DBMS tools. Some standard Entando database schemas to back up are `prod_dedb`, `prod_kc_db`, `prod_portdb`, and `prod_servdb`, in addition to any that were created for your microservices. Those will follow a similar naming convention, with the microservice name followed by the suffix `_plugindb`.
       * If the original database setup used a **custom resource**, refer to the [Database Custom Resource](https://developer.entando.com/v7.1/docs/reference/database-cr.html) document. 
       * If it was provisioned using **environment variables** (PORTDB_*, SERVDB_* variables) in the EntandoApp custom resource, see the [External DB](https://developer.entando.com/v7.1/tutorials/devops/external-db.html) document for more information.  
  
-2. Restore the backups to the final DBMS.
-3. Install the restored instance by copying the data as needed.
+2. Restore the backups to the target DBMS.
+
 
 ## Backup and Extract the AppEngine Filesystem Data
 Use tar commands to archive and extract the files, with kubectl providing support.
-1. Use a temporary directory to store the tarball of the data files for the AppEngine pod `quickstart-deployment`:
+1. Shell in to the `quickstart-deployment` pod as the tar commands need to run from inside of the pod.
```

Comment:
pod, (without the comma the reader expects conditions/requirements for shelling in, like you can shell in as an admin, etc. the comma sets an expectation that what follows is not a modification of that action)

"to run inside the pod" is more succinct, no meaning changed or details lost